modules/stitching/include/opencv2/stitching.hpp

```diff
@@ -193,6 +193,9 @@ class CV_EXPORTS_W Stitcher
     CV_WRAP bool waveCorrection() const { return do_wave_correct_; }
     CV_WRAP void setWaveCorrection(bool flag) { do_wave_correct_ = flag; }
 
+    CV_WRAP int interpolation() const { return interp_mode_; }
+    CV_WRAP void setInterpolation(int interp) { interp_mode_ = interp; }
```

Comment:
> int

It is better to use `InterpolationFlags` enum instead of raw "int".

Reply:
Indeed, this is fixed